addons/markers/stringtable.xml

```diff
@@ -24,6 +24,38 @@
             <Chinese>方位: %1°</Chinese>
             <Turkish>Yön: %1°</Turkish>
         </Key>
+        <Key ID="STR_ACE_Markers_MarkerDirectionSlider_Tooltip">
+            <English>Click RMB to reset to 0</English>
+            <German>Click RMB to reset to 0</German>
+            <Japanese>Click RMB to reset to 0</Japanese>
+            <Italian>Click RMB to reset to 0</Italian>
+            <Chinese>Click RMB to reset to 0</Chinese>
+            <Chinesesimp>Click RMB to reset to 0</Chinesesimp>
+            <Korean>Click RMB to reset to 0</Korean>
+            <Polish>Click RMB to reset to 0</Polish>
+            <Russian>Кликните ПКМ чтобы сбросить на 0</Russian>
+            <Portuguese>Click RMB to reset to 0</Portuguese>
+            <French>Click RMB to reset to 0</French>
+            <Czech>Click RMB to reset to 0</Czech>
+            <Turkish>Click RMB to reset to 0</Turkish>
+            <Spanish>Click RMB to reset to 0</Spanish>
+        </Key>
+        <Key ID="STR_ACE_Markers_MarkerScaleSlider_Tooltip">
+            <English>Click RMB to reset to 1</English>
+            <German>Click RMB to reset to 1</German>
+            <Japanese>Click RMB to reset to 1</Japanese>
+            <Italian>Click RMB to reset to 1</Italian>
+            <Chinese>Click RMB to reset to 1</Chinese>
+            <Chinesesimp>Click RMB to reset to 1</Chinesesimp>
+            <Korean>Click RMB to reset to 1</Korean>
+            <Polish>Click RMB to reset to 1</Polish>
+            <Russian>Кликните ПКМ чтобы сбросить на 1</Russian>
+            <Portuguese>Click RMB to reset to 1</Portuguese>
+            <French>Click RMB to reset to 1</French>
+            <Czech>Click RMB to reset to 1</Czech>
+            <Turkish>Click RMB to reset to 1</Turkish>
+            <Spanish>Click RMB to reset to 1</Spanish>
+        </Key>
```

Comment:
please keep unlocalized languages empty, so they are getting correctly picked up by #367.

I also think that `Click RMB to reset` should be enough in this case

Reply:
Fixed in https://github.com/acemod/ACE3/pull/8319/commits/b0522b37c7675e8cd50fc2ec9986ee8f58796e5b